unikernel_create.ml

```diff
@@ -16,37 +21,184 @@ let unikernel_create_layout =
-                  [ txt "Name*" ];
-                input
+                div
+                  [
+                    label ~a:[ a_class [ "block font-medium" ] ] [ txt "Name*" ];
+                    input
+                      ~a:
+                        [
+                          a_input_type `Text;
+                          a_name "name";
+                          a_required ();
+                          a_id "unikernel-name";
+                          a_class
+                            [
+                              "ring-primary-100 mt-1.5 transition \
+                               appearance-none block w-full px-3 py-3 \
+                               rounded-xl shadow-sm border \
+                               hover:border-primary-200\n\
+                              \                                           \
+                               focus:border-primary-300 bg-primary-50 \
+                               bg-opacity-0 hover:bg-opacity-50 \
+                               focus:bg-opacity-50 ring-primary-200 \
+                               focus:ring-primary-200\n\
+                              \                                           \
+                               focus:ring-[1px] focus:outline-none";
+                            ];
+                        ]
+                      ();
+                  ];
+                div
+                  [
+                    label
+                      ~a:[ a_class [ "block font-medium" ] ]
+                      [ txt "CPU Id" ];
+                    select
+                      ~a:
+                        [
+                          a_id "cpuid";
+                          a_name "cpuid";
+                          a_class
+                            [
+                              "ring-primary-100 mt-1.5 transition block w-full \
+                               px-3 py-3 rounded-xl shadow-sm border \
+                               hover:border-primary-200\n\
+                              \                                           \
+                               focus:border-primary-300 bg-primary-50 \
+                               bg-opacity-0 hover:bg-opacity-50 \
+                               focus:bg-opacity-50 ring-primary-200 \
+                               focus:ring-primary-200\n\
+                              \                                           \
+                               focus:ring-[1px] focus:outline-none";
+                            ];
+                        ]
+                      (List.map
+                         (fun (cpu_id, count) ->
+                           option
+                             ~a:[ a_value (string_of_int cpu_id) ]
+                             (txt
+                                ("CPU " ^ string_of_int cpu_id ^ " (used by "
+                               ^ string_of_int count ^ " unikernels)")))
+                         count_cpuid_usage);
+                  ];
```

Comment:
I'm not a HTML (form) expert, but should we by default have a selection that is the least used CPUid? Thus, by default nobody needs to click anything ;)